### PR TITLE
msrest 0.4.3 + msrestazure 0.4.2

### PR DESCRIPTION
**Pull request by lmazuel**
Release client runtime of this PR: https://github.com/Azure/autorest/pull/1392

https://pypi.python.org/pypi/msrestazure/0.4.2
https://pypi.python.org/pypi/msrest/0.4.3

### CODE REVIEW

File: src/client/Python/msrest/readme.rst [hunk 1]
```diff
@@ -15,6 +15,13 @@ To install:
 Release History
 ---------------
 
+2016-09-01 Version 0.4.3
+++++++++++++++++++++++++
+
+**Bugfixes**
+
+- Better exception message (https://github.com/Azure/autorest/pull/1300)
```
Comment (yugangw-msft) [inline src/client/Python/msrest/readme.rst]: should we point to the [issue](https://github.com/Azure/azure-sdk-for-python/issues/592), rather this pull request, which is still active?

Comment (yugangw-msft) [inline src/client/Python/msrest/readme.rst]: also the fix is in msrestazure, not the generic msrest, so we probably should mention in msrestazure's readme

Comment (yugangw-msft) [inline src/client/Python/msrest/readme.rst]: ah, looks like you did it already, so ignoring my previous comment

**Comment by yugangw-msft** [comment]
the rest lgtm

**Comment by tbombach** [comment]
Merged

**Pull request merged**